app/Resources/views/base.html.twig

```diff
@@ -5,9 +5,9 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>
             {% if gmTitle is defined %}{{ msg_if_exists(gmTitle) }} - {% elseif exception is defined %}{{ msg('fatal-error') }} - {% endif %}
-            {{ msg('grant-metrics') }}
+            {{ msg('event-metrics') }}
         </title>
-        <link rel="icon" type="image/x-icon" sizes="16x16 32x32 48x48" href="{{ asset('favicon.ico') }}" />
+        <link rel="icon" type="image/x-icon" sizes="16x16 32x32 48x48" href="{{ asset('favicon.ico') }}?v=2" />
```

Comment:
Cache-busting could be handled by webpack like other assets are, if this file was in the assets directory and something `require()`'d it.

Probably a separate issue though.

Reply:
e.g. here: https://github.com/wikimedia/svgtranslate/blob/59c41227cefe02cf7b7e77b5b60e84e21b3c1e18/assets/app.js#L11